sdk/src/main/java/com/silanis/esl/sdk/internal/converter/DocumentPackageSettingsConverter.java

```diff
@@ -185,6 +187,9 @@ public DocumentPackageSettingsConverter(com.silanis.esl.sdk.DocumentPackageSetti
         if (apiPackageSettings.getCeremony().getFontSize() != null)
             result.setFontSize(apiPackageSettings.getCeremony().getFontSize());
 
+        result.setDefaultTimeBasedExpiry(apiPackageSettings.getCeremony().getDefaultTimeBasedExpiry());
+        result.setRemainingDays(apiPackageSettings.getCeremony().getRemainingDays());
```

Comment:
Check whether it is null or not. If you don't set these values, it may be updated to null when updated.